source/rules/prefer-observer.ts

```diff
@@ -24,8 +24,8 @@ const rule = ruleCreator({
         "Forbids the passing separate handlers to `subscribe` and `tap`.",
       recommended: false,
     },
-    fixable: undefined,
-    hasSuggestions: false,
+    fixable: "code",
```

Comment:
Honestly not totally sure how this works. Ideally, people should be able to run this with `--fix` and fix their whole code base. I'd like to try this on the RxJS tests because there are hundreds of legacy calls we need to fix in there.